conda/environment.yml

```diff
@@ -3,9 +3,8 @@ channels:
   - conda-forge
 dependencies:
   # runtime dependencies
-  - apache-flink
+  # - apache-flink
```

Comment:
Ups, my custom environment fixes bled into this commit, inadvertently. Good thing, i had to revert it.